inductiva/simulators/simulator.py

```diff
@@ -11,6 +13,7 @@ class Simulator(ABC):
 
     def __init__(self):
         self.api_method_name = ""
+        self.checked_resource_pool = False
```

Comment:
In principle, this can be a class attribute, hence when its value changes when you do `sim = Simulator()` it will be changed for further initializations in the "session".